python/lsst/sims/photUtils/PhotometricParameters.py

```diff
@@ -1,7 +1,37 @@
 from builtins import object
-import numpy
+import numpy as np
+from lsst.sims.photUtils import Sed, Bandpass
+
+__all__ = ["PhotometricParameters", 'Dust_values']
+
+
+class Dust_values(object):
+    """Calculate extinction values
```

Comment:
Need to explicitly state which dust values we're calculating, to avoid ambiguity. There are at least two different dust extinction laws floating around.